moncli/entities/column_value/simple.py

```diff
@@ -117,7 +117,46 @@ class PhoneValue(ComplexNullValue):
 
 class StatusValue(ComplexNullValue):
     """A status column value."""
+    native_type = str
+    allow_casts = (int, str)
 
+    def _convert(self, value):
+        settings = self.settings
+        labels = settings['labels']
+        index = str(value['index'])
+        value = labels[index]  
+        return value
+    
+    def _cast(self, value):
+        if isinstance(value,int):
+            index = str(value)
+            labels = self.settings['labels']
+            try:
+                value = labels[index]
+                return value
+            except KeyError:
+                raise ColumnValueError( 'invalid_status_index',self.id )
+        if isinstance(value,str):
+            labels = self.settings['labels']
+            try:
+                int_value = int(value)
+                label = labels[value]
+                return label
+            except ValueError:
+                if value in labels.values():
+                    return value
+                raise ColumnValueError( 'invalid_status_index',self.id,
+                                        'Cannot find a status with the following index "{}".format(value)' )
+        
+    def _format(self):
+        if self.value == None:
```

Comment:
Luckily, this case is already handled by the base class before it gets to this point.  You may go ahead and remove this.